plugin_tests/widgetSpec.js

```diff
@@ -0,0 +1,644 @@
+girderTest.importPlugin('jobs', 'worker', 'slicer_cli_web');
```

Comment:
Other than indentation, I assume nothing else is changed beyond the first 8 lines?